<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaitWindow</class>
 <widget class="QMainWindow" name="FaitWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>11</y>
      <width>541</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2" colspan="2">
      <widget class="QTextEdit" name="Textbatlle"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="btn_ataka">
       <property name="text">
        <string>Атака</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTextEdit" name="MonstertextEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QTextEdit" name="PlayertextEdit"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="def">
       <property name="text">
        <string>Защита</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="bag">
       <property name="text">
        <string>тут нужно поместить рюкзак</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="miss">
       <property name="text">
        <string>Уклонение</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="battle">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать бой</string>
    </property>
   </widget>
   <widget class="QPushButton" name="go">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Идти дальше</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
